google-cloud-recaptcha_enterprise/samples/acceptance/data/test_template.html

```diff
@@ -0,0 +1,190 @@
+<!--
+
```

Comment:
Are all these apparently doubled newlines because they need to be `\r\n`? Maybe consider using standard `\n` newlines and then gsubbing them in the test code. Otherwise it might be difficult for the next engineer editing this file to preserve the newlines properly.

Reply:
yes they should be good just with `\n`, fixed this 👍